install/mysql/glpi-empty.sql

```diff
@@ -9623,6 +9623,28 @@ CREATE TABLE `glpi_pdus_racks` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
 -- /Datacenters
 
+DROP TABLE IF EXISTS `glpi_itilfollowuptemplates`;
+CREATE TABLE `glpi_itilfollowuptemplates` (
+   `id`              INT(11) NOT NULL AUTO_INCREMENT,
+   `date_creation`   TIMESTAMP NULL DEFAULT NULL,
+   `date_mod`        TIMESTAMP NULL DEFAULT NULL,
+   `entities_id`     INT(11) NOT NULL DEFAULT '0',
+   `is_recursive`    TINYINT(1) NOT NULL DEFAULT '0',
+   `name`            VARCHAR(255) NULL DEFAULT NULL COLLATE 'utf8_unicode_ci',
+   `content`         TEXT NULL COLLATE 'utf8_unicode_ci',
+   `requesttypes_id` INT(11) NOT NULL DEFAULT '0',
+   `is_private`      TINYINT(1) NOT NULL DEFAULT '0',
+   `comment`         TEXT NULL COLLATE 'utf8_unicode_ci',
+   PRIMARY KEY (`id`),
+   INDEX `name` (`name`),
```

Comment:
Why not `KEY` as usual?

Reply:
i think i used heidisql to do the export, will fixed